fmriprep/workflows/bold/registration.py

```diff
@@ -456,39 +464,25 @@ def init_bbreg_wf(use_bbr, bold2t1w_dof, omp_nthreads, name='bbreg_wf'):
                     generate_report=not use_bbr),
         name='mri_coreg', n_procs=omp_nthreads, mem_gb=5)
 
-    lta_concat = pe.Node(ConcatenateLTA(out_file='out.lta'), name='lta_concat')
-    # XXX LTA-FSL-ITK may ultimately be able to be replaced with a straightforward
-    # LTA-ITK transform, but right now the translation parameters are off.
-    lta2fsl_fwd = pe.Node(LTAConvert(out_fsl=True), name='lta2fsl_fwd')
-    lta2fsl_inv = pe.Node(LTAConvert(out_fsl=True, invert=True), name='lta2fsl_inv')
-    fsl2itk_fwd = pe.Node(c3.C3dAffineTool(fsl2ras=True, itk_transform=True),
-                          name='fsl2itk_fwd', mem_gb=DEFAULT_MEMORY_MIN_GB)
-    fsl2itk_inv = pe.Node(c3.C3dAffineTool(fsl2ras=True, itk_transform=True),
-                          name='fsl2itk_inv', mem_gb=DEFAULT_MEMORY_MIN_GB)
+    merge_ltas = pe.Node(niu.Merge(2), name='merge_ltas', run_without_submitting=True)
+    concat_xfm = pe.Node(ConcatenateXFMs(inverse=True), name='concat_xfm')
 
     workflow.connect([
         (inputnode, mri_coreg, [('subjects_dir', 'subjects_dir'),
                                 ('subject_id', 'subject_id'),
                                 ('in_file', 'source_file')]),
         # Output ITK transforms
-        (inputnode, lta_concat, [('fsnative2t1w_xfm', 'in_lta2')]),
-        (lta_concat, lta2fsl_fwd, [('out_file', 'in_lta')]),
-        (lta_concat, lta2fsl_inv, [('out_file', 'in_lta')]),
-        (inputnode, fsl2itk_fwd, [('t1w_brain', 'reference_file'),
-                                  ('in_file', 'source_file')]),
-        (inputnode, fsl2itk_inv, [('in_file', 'reference_file'),
-                                  ('t1w_brain', 'source_file')]),
```

Comment:
These were performing the work of ensuring that the ITK-orientation (qform) was being used in the conversion. I guess we're ensuring that `qform == sform` and that's all encoded in the LTA?

Reply:
Sorry I don't know how to address the question, so I'll go step by step and then see if that was what you wanted to clarify.

1. LTA does include q/s-form information in the file, and it is necessary to interpret the transform only if it is VOX2VOX type.
2. NiTransforms converts all LTAs with VOX2VOX type into RAS2RAS type so that the s/q-form information is detached from the transform (i.e., it operates in physical coordinates).
3. NiTransforms writes an ITK transform. ITK transforms are completely(*) unaware of the notion of the images' orientation. (*) this is not 100% true for the case of displacements fields, but this is not the case here.
4. These connections enforced observing q/s-forms because FSL embeds that information within the transform.

Did that get at the bottom of your point?

Reply:
Ah, right. It's FSL that does a VOX2VOX. I think we're good. Thanks.